src/nvim/version.c

```diff
@@ -1027,11 +1027,11 @@ void intro_message(int colon)
   };
 
   // blanklines = screen height - # message lines
-  blanklines = (int)Rows - (ARRAY_SIZE(lines) - 1);
+  blanklines = (int)Rows - (int)(ARRAY_SIZE(lines) - 1);
```

Comment:
As the introduction message is small, the number of lines of this message will also be small.